### ISSUE TITLE
Braille display doesn't work in NVDA 2022.1

### ISSUE THREAD

**Issue by AppleUser03** [lambda-nvda/lambdaNvda#58]
I use NVDA 2022.1 and the version 1.4.0 of the addon. When I open Lambda, the synthesizer works correctly, but the Braille display remains blank, so I can't read anything on it. How can I fix it?

**Comment by ivnc** [comment]
Reproducible here. Maybe caused by Liblouis update introduced with 2022.1? @albzan
ERROR - eventHandler.executeEvent (18:33:37.537) - MainThread (3968):
error executing event: gainFocus on <NVDAObjects.IAccessible.MenuItem object at 0x036E26F0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 284, in executeEvent
  File "eventHandler.pyc", line 98, in __init__
  File "eventHandler.pyc", line 107, in next
  File "appModules\nvda.pyc", line 208, in event_gainFocus
  File "eventHandler.pyc", line 107, in next
  File "NVDAObjects\IAccessible\__init__.pyc", line 2034, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 1153, in event_gainFocus
  File "braille.pyc", line 2017, in handleGainFocus
  File "braille.pyc", line 2022, in _doNewObject
  File "braille.pyc", line 1641, in getFocusContextRegions
  File "braille.pyc", line 664, in update
  File "braille.pyc", line 442, in update
  File "louisHelper.pyc", line 65, in translate
  File "louis\__init__.pyc", line 267, in translate
RuntimeError: Can't translate: tables ['C:\\Users\\inovegil\\AppData\\Roaming\\nvda\\addons\\lambda\\appModules\\lambda\\brailleTables\\lambda-esp.utb', 'braille-patterns.cti'], inbuf b'F\x00\x00\x00e\x00\x00\x00r\x00\x00\x00r\x00\x00\x00a\x00\x00\x00m\x00\x00\x00e\x00\x00\x00n\x00\x00\x00t\x00\x00\x00a\x00\x00\x00s\x00\x00\x00 \x00\x00\x00m\x00\x00\x00n\x00\x00\x00 \x00\x00\x00', typeform None, cursorPos c_long(0), mode 4

**Comment by albzan** [comment]
Yes, my fault. I've fixed this but I didn't compiled after the bug fix. To let user update I'll release a new version, which has nothing new except a rebuild.

**Comment by albzan** [comment]
Hi,
Can you try this and let me know if it has been fixed?
https://github.com/lambda-nvda/lambdaNvda/releases/download/1.4.1/lambda-1.4.1.nvda-addon

**Comment by AppleUser03** [comment]
Thank you Alberto and Ivan, all is fixed! You only had to publish this version so all user can download it with Addon Updater. If I can make a suggestion: is it possible to use these custom Braille tables also for writing and not only for reading in a future version of the addon?

**Comment by albzan** [comment]
Yes good point! Thank you! I'll keep track of it for the next release
